### PR TITLE
correct typo

### PR DESCRIPTION
**Pull request by cedrict**
correcting a simple typo and adding a folder location in the manual section about creating tests.

### CODE REVIEW

File: doc/manual/manual.tex [hunk 1]
```diff
@@ -11868,13 +11868,13 @@ \subsubsection{Writing tests}
...
 \begin{lstlisting}[frame=single,language=ksh]
-    make setup\_tests
+    make setup_tests
 \end{lstlisting}
 so that your new test is added to the test suite. Then you can run it by executing
 \begin{lstlisting}[frame=single,language=ksh]
     ctest -R name_of_your_test -V
 \end{lstlisting}
-and you will get an output telling you if the test has
+in the \texttt{tests} folder and you will get an output telling you if the test has
```
Comment (anne-glerum) [inline doc/manual/manual.tex:11877]: But you don't run the ctest command in the test folder, but in the build folder, right?

**Review by anne-glerum — CHANGES_REQUESTED** [review]

File: doc/manual/manual.tex (hunk 1 above)
Comment (cedrict) [reply doc/manual/manual.tex:11877]: so ... when I wrote this it did not work for me, only in the tests folder. Of course, now, fresh install, it works as intended in the build folder :(

**Review by anne-glerum — APPROVED** [review]
Glad it works for you now!

**Pull request merged**